sparrow/sparrow.md

```diff
@@ -18,8 +18,49 @@ nav_order: 70
 ---
 ### Overview
 
-[Sparrow](https://www.sparrowwallet.com/){:target="_blank" rel="noopener"} is an open-source Bitcoin wallet focused on security, privacy, and usability. It provides detailed, user-friendly information about transactions and UTXOs, supporting financial self-sovereignty. 
+[Sparrow] is an open-source Bitcoin wallet focused on security, privacy, and usability. It provides detailed, user-friendly information about transactions and UTXOs, supporting financial self-sovereignty. 
+![sparrow.png](sparrow.png)
 
+---
+### Install the latest version of Sparrow
+
+* Open a _Terminal_ window:  choose **Applications ▸ Utilities ▸ Terminal**
+
+* Clone Run-on-Tails GitHub repository:
+```shell
+$ cd ~/Downloads
+$ git clone https://github.com/dutu/run-on-tails.git
+```
+
+* Setup Sparrow:
+  ```shell
+  $ chmod +x ./run-on-tails/electrum/setup-installation.sh 
+  $ ./run-on-tails/electrum/setup-installation.sh 
+  ```
+  * Wait for the message `Sparrow installation setup completed successfully.`
+
+ ---
+### How to use it
+
+* Choose **Applications ▸ Other ▸ Electrum**
+
+
+---
+### Backup your wallet
+
+* While Sparrow is closed, copy your Sparrow wallet directory located at `/home/amnesia//Persistent/Sparrow/data/wallets` to a backup location.
+
+
+---
+### Remove the latest version of Sparrow
+
+* Open a _Terminal_ window:  choose **Applications ▸ Utilities ▸ Terminal**
 
-Coming soon
-{: .label .label-yellow }
\ No newline at end of file
+
+* Remove Electrum application files:
```

Comment:
I see the `wallets` directory is also removed. Would it be possible to leave it and remove the application files only?